README.md

```diff
@@ -0,0 +1,193 @@
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/engines.html">Embbed Ruby (ERB)</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Log into the application**
+- **Create a category**
+- **Add transaction**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+### Live demo 
+
+Here is a live [demo](https://budgetracker.onrender.com) of the application
+
+### Video presentation
+Here is the link to the video presentation of the [project](https://www.loom.com/share/1e0b6607108c4e4fb2f9d6ea54bd8cb7)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+ - Ruby and rails installed
+- PostgreSQL server installed also
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  https://github.com/anyepeter/rails-budget.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd rails-budget
+  bundle install 
+  rails db:create
+  rails db:schema:load
+```
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **ANYE PETER**
+
+- GitHub: [@anyepeter](https://github.com/anyepeter)
+- Twitter: [@anyepeter](https://twitter.com/home?lang=en)
+- LinkedIn: [peterchendi](https://www.linkedin.com/feed/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **A better graphical user interface**
+- [ ] **Add a credit card**
+- [ ] **A chat functionality**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, go ahead and give it a star. It is very much appreciated.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+- I would like to thank Microvers for providing this great opportunity and for the project guide as well 👏
+- Huge appriciation to Gregoire Vella for providing design system guidelines since this project is built upon thatopment.
```

Comment:
- [ ] [ OPTIONAL ] please make sure to correct the typing errors in this sentence like `appriciation and thatopment`